src/OSPSuite.Infrastructure.Import/Services/ICsvSeparatorSelector.cs

```diff
@@ -1,16 +1,10 @@
 ﻿namespace OSPSuite.Infrastructure.Import.Services
 {
-   public class CSVSeparators
-   {
-      public char ColumnSeparator { get; set; }
-      public char DecimalSeparator { get; set; }
-   }
-
    public interface ICsvSeparatorSelector
    {
       /// <summary>
       ///    Returns the separator to be used for the reading of the .csv file
       /// </summary>
-      CSVSeparators GetCsvSeparator(string fileName);
+      (char? ColumnSeparator, char DecimalSeparator) GetCsvSeparator(string fileName);
```

Comment:
Naming is off I would say. When you retrieve then, they should be like local variables no?
So columnSeparator and decimalSeparator